Some CI jobs are failing with the same error. This seems like an issue with the tooling. @lirenhe @zikalino @yungezz could you please take a look?

```
scripts/multiapi.ts(95,40): error TS2345: Argument of type 'Set<string>' is not assignable to parameter of type 'Iterable<any>'.
  Types of property '[Symbol.iterator]' are incompatible.
    Type '() => IterableIterator<string>' is not assignable to type '() => Iterator<any>'.
      Type 'IterableIterator<string>' is not assignable to type 'Iterator<any>'.
        Types of property 'next' are incompatible.
          Type '(...args: [] | [undefined]) => IteratorResult<string, any>' is not assignable to type '() => IteratorResult<any>'.
            Type 'IteratorResult<string, any>' is not assignable to type 'IteratorResult<any>'.
              Type 'IteratorYieldResult<string>' is not assignable to type 'IteratorResult<any>'.
                Types of property 'done' are incompatible.
                  Type 'false | undefined' is not assignable to type 'boolean'.
                    Type 'undefined' is not assignable to type 'boolean'.
npm ERR! Test failed.  See above for more details.
```

--- a/specification/network/resource-manager/Microsoft.Network/stable/2019-07-01/examples/VirtualNetworkGatewayConnectionCreate.json
+++ b/specification/network/resource-manager/Microsoft.Network/stable/2019-07-01/examples/VirtualNetworkGatewayConnectionCreate.json
@@ -61,7 +61,8 @@
         "sharedKey": "Abc123",
         "enableBgp": false,
         "usePolicyBasedTrafficSelectors": false,
-        "ipsecPolicies": []
+        "ipsecPolicies": [],
+        "trafficSelectorPolicies": []
       },
       "location": "centralus"
     }

--- a/specification/network/resource-manager/Microsoft.Network/stable/2019-07-01/examples/VirtualNetworkGatewayConnectionGet.json
+++ b/specification/network/resource-manager/Microsoft.Network/stable/2019-07-01/examples/VirtualNetworkGatewayConnectionGet.json
@@ -31,6 +31,7 @@
           "enableBgp": false,
           "usePolicyBasedTrafficSelectors": false,
           "ipsecPolicies": [],
+          "trafficSelectorPolicies": [],
           "connectionStatus": "Connecting",
           "ingressBytesTransferred": 0,
           "egressBytesTransferred": 0

--- a/specification/network/resource-manager/Microsoft.Network/stable/2019-07-01/examples/VirtualNetworkGatewayConnectionUpdateTags.json
+++ b/specification/network/resource-manager/Microsoft.Network/stable/2019-07-01/examples/VirtualNetworkGatewayConnectionUpdateTags.json
@@ -40,6 +40,7 @@
           "enableBgp": false,
           "usePolicyBasedTrafficSelectors": false,
           "ipsecPolicies": [],
+          "trafficSelectorPolicies": [],
           "connectionStatus": "Unknown",
           "ingressBytesTransferred": 0,
           "egressBytesTransferred": 0

--- a/specification/network/resource-manager/Microsoft.Network/stable/2019-07-01/examples/VirtualNetworkGatewayConnectionsList.json
+++ b/specification/network/resource-manager/Microsoft.Network/stable/2019-07-01/examples/VirtualNetworkGatewayConnectionsList.json
@@ -31,6 +31,7 @@
               "enableBgp": false,
               "usePolicyBasedTrafficSelectors": false,
               "ipsecPolicies": [],
+              "trafficSelectorPolicies": [],
               "ingressBytesTransferred": 0,
               "egressBytesTransferred": 0
             }
@@ -58,6 +59,7 @@
               "enableBgp": false,
               "usePolicyBasedTrafficSelectors": false,
               "ipsecPolicies": [],
+              "trafficSelectorPolicies": [],
               "ingressBytesTransferred": 0,
               "egressBytesTransferred": 0
             }

--- a/specification/network/resource-manager/Microsoft.Network/stable/2019-07-01/examples/VirtualNetworkGatewaysListConnections.json
+++ b/specification/network/resource-manager/Microsoft.Network/stable/2019-07-01/examples/VirtualNetworkGatewaysListConnections.json
@@ -29,6 +29,7 @@
               "enableBgp": true,
               "usePolicyBasedTrafficSelectors": false,
               "ipsecPolicies": [],
+              "trafficSelectorPolicies": [],
               "ingressBytesTransferred": 0,
               "egressBytesTransferred": 0
             }

--- a/specification/network/resource-manager/Microsoft.Network/stable/2019-07-01/virtualNetworkGateway.json
+++ b/specification/network/resource-manager/Microsoft.Network/stable/2019-07-01/virtualNetworkGateway.json
@@ -2370,6 +2370,13 @@
           },
           "description": "The IPSec Policies to be considered by this connection."
         },
+        "trafficSelectorPolicies": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/TrafficSelectorPolicy"
+          },
+          "description": "The Traffic Selector Policies to be considered by this connection."
+        },
         "resourceGuid": {
           "type": "string",
           "description": "The resource GUID property of the virtual network gateway connection resource."
@@ -2509,6 +2516,29 @@
         "pfsGroup"
       ],
       "description": "An IPSec Policy configuration for a virtual network gateway connection."
+    },
+    "TrafficSelectorPolicy": {
+      "properties": {
+        "localAddressRanges": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "A collection of local address spaces in CIDR format"
+        },
+        "remoteAddressRanges": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "A collection of remote address spaces in CIDR format"
+        }
+      },
+      "required": [
+        "localAddressRanges",
+        "remoteAddressRanges"
+      ],
+      "description": "An traffic selector policy for a virtual network gateway connection."
     },
     "ConnectionProtocol": {
       "type": "string",
@@ -2726,6 +2756,13 @@
             "$ref": "#/definitions/IpsecPolicy"
           },
           "description": "The IPSec Policies to be considered by this connection."
+        },
+        "trafficSelectorPolicies": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/TrafficSelectorPolicy"
+          },
+          "description": "The Traffic Selector Policies to be considered by this connection."
         },
         "resourceGuid": {
           "type": "string",

